teaser/examples/verification/verification_ASHRAE_140_920.py

```diff
@@ -65,7 +65,7 @@ def from_scratch(number_of_elements, save=False, path=utilities.get_default_path
         Project that contains the building with the test room
 
     """
-    prj = Project(load_data=True)
+    prj = Project(load_data=False)
```

Comment:
Consider removing all `load_data=False` and just call `Project()`, as load_data has no effect, oir?